SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/OpsUtils.h

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+inline TArray<OpList> ConstructOpList(TArray<Worker_Op>& Ops)
```

Comment:
Name seems to imply this would just return a single oplist. Should it still be wrapped in an array?

Reply:
turns out: There is already an op list builder in the repo. Gonna get rid of the utils here and extend the builder to what we need